src/localSettings.ts

```diff
@@ -0,0 +1,15 @@
+// NEVER EVER COMMIT THIS FILE!!

```

Comment:
Damn, the way this message screams... Makes u think the world will explode if you EVER commit this file... :)